ci/Jenkinsfile

```
@@ -179,6 +183,14 @@ pipeline {
   }
 }
 
+def addAllureCategories() {
+    sh "cp ${'ext/allure_files/categories.json'} ${'allure-results'}"
```

Why? I don't get what's the point of using string interpolation here.
```suggestion
    sh 'cp ext/allure_files/categories.json allure-results'
```

---

Fixed